### PR TITLE
Package loky

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/staged-recipes/issues/6063

### CODE REVIEW

File: recipes/loky/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
+  license: BSD
+  # No license yet
+  # license_file:
```
Comment (jakirkham) [inline recipes/loky/meta.yaml]: Currently there isn't a license file. Raised issue ( https://github.com/tomMoral/loky/issues/136 ) to add one and get it packaged.

File: recipes/loky/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+    - cloudpickle
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
+  license: BSD
```
Comment (jakirkham) [inline recipes/loky/meta.yaml]: Guessing 3-Clause, but am a little unclear. Hopefully will be clear after the license file issue is addressed.

**Comment by jakirkham** [comment]
@tomMoral, would appreciate your thoughts on this. Also if you have any interest in being a maintainer, would be happy to add you. Thanks.

File: recipes/loky/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+    - cloudpickle
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
```
Comment (tomMoral) [inline recipes/loky/meta.yaml]: I don't know the standard for `conda` packages but would it be better to link to the documentation (http://loky.readthedocs.io) for the home page?

File: recipes/loky/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
+  summary: A robust implementation of concurrent.futures.ProcessPoolExecutor
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```
Comment (tomMoral) [inline recipes/loky/meta.yaml]: You can add me in the maintainers of the recipe, so I can help update when we release new version.

File: recipes/loky/meta.yaml (hunk 3 above)
Comment (jakirkham) [reply recipes/loky/meta.yaml]: We use whatever is specified as the [`url` on PyPI]( https://github.com/tomMoral/loky/blob/2.1.4/setup.py#L60 ) for `home`. That said, have added the `doc_url` and `dev_url` fields. The former now points to the RTD page.

File: recipes/loky/meta.yaml (hunk 4 above)
Comment (jakirkham) [reply recipes/loky/meta.yaml]: Cool, thanks. Have added you.

**Comment by jakirkham** [comment]
Thoughts @conda-forge/staged-recipes?

**Comment by mwcraig** [comment]
LGTM, merging...

**Pull request merged**